build/gulpfile.extensions.js

```diff
@@ -285,7 +285,7 @@ gulp.task(packageLocalizationExtensionsTask);
 const compileLocalizationExtensionsBuildTask = task.define('compile-localization-extensions-build', task.series(
 	cleanExtensionsBuildTask,
 	compileExtensionsTask,
-	task.define('bundle-marketplace-extensions-build', () => ext.packageMarketplaceExtensionsStream(false, product.extensionsGallery?.serviceUrl).pipe(gulp.dest('.build'))),
+	task.define('bundle-marketplace-extensions-build', () => { ext.packageMarketplaceExtensionsStream(false, product.extensionsGallery?.serviceUrl).pipe(gulp.dest('.build')) }),
```

Comment:
This is changing the behavior by not returning the value from packageMarketplaceExtensionsStream. Doing this caused breaking issues in the last merge (fixed by https://github.com/microsoft/azuredatastudio/pull/21117) so likely would do the same here.